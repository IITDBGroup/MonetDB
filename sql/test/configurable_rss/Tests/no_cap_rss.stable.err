stderr of test 'no_cap_rss` in directory 'sql/test/configurable_rss` itself:


# 20:52:50 >  
# 20:52:50 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=31687" "--set" "monet_prompt=" "--forcemito" "--dbpath=C:\Users\aris\monetdb-installation\var\MonetDB\mTests_sql_test_configurable_rss"
# 20:52:50 >  

# builtin opt 	gdk_dbpath = C:\Users\aris\monetdb-installation\var\monetdb5\dbfarm\demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 31687
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = C:\Users\aris\monetdb-installation\var\MonetDB\mTests_sql_test_configurable_rss
# cmdline opt 	gdk_debug = 553648138

# 20:52:51 >  
# 20:52:51 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=DESKTOP-J5OE0E3" "--port=31687"
# 20:52:51 >  


# 20:53:00 >  
# 20:53:00 >  "Done."
# 20:53:00 >  

