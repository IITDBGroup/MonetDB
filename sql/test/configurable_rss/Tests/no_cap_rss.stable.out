stdout of test 'no_cap_rss` in directory 'sql/test/configurable_rss` itself:


# 20:52:50 >  
# 20:52:50 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=31687" "--set" "monet_prompt=" "--forcemito" "--dbpath=C:\Users\aris\monetdb-installation\var\MonetDB\mTests_sql_test_configurable_rss"
# 20:52:50 >  

# MonetDB 5 server v11.30.0
# This is an unreleased version
# Serving database 'mTests_sql_test_configurable_rss', using 2 threads
# Compiled for x86_64-pc-winnt/64bit
# Found 8.000 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2018 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://DESKTOP-J5OE0E3:31687/
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 20:52:51 >  
# 20:52:51 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=DESKTOP-J5OE0E3" "--port=31687"
# 20:52:51 >  

#create function getrss() 
#returns bigint external name status.rss_cursize;
#create function getvm() 
#returns bigint external name status.mem_cursize;
#create table test(a int, b int, c double);
#insert into test values (1, 0, 1);
[ 1	]
#create procedure loop_insert(maximum_size bigint)
#begin
#    declare size bigint;
#    set size = (select count(*) from test);
#
#    while size < maximum_size do
#        insert into test (select a+1, b+2, rand()*c from test);
#
#        set size = (select count(*) from test);
#    end while;
#end;
#select count(*) as record_count from test;
% sys.L4 # table_name
% record_count # name
% bigint # type
% 8 # length
[ 16777216	]
#select getrss() > 70000 as resident_set_size_is_bigger_then_70kbytes;
% .L2 # table_name
% resident_set_size_is_bigger_then_70kbytes # name
% boolean # type
% 5 # length
[ true	]
#drop table test cascade;
#drop function getrss;
#drop function getvm;

# 20:53:00 >  
# 20:53:00 >  "Done."
# 20:53:00 >  

